src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -29,83 +29,80 @@ function Test-StorageAccount
-        Assert-AreEqual $sto.Encryption.Services.File.Enabled $true
+        Assert-AreEqual $stoname $sto.StorageAccountName;
+        Assert-AreEqual $stotype $sto.Sku.Name;
+        Assert-AreEqual $loc $sto.Location;
+        Assert-AreEqual $kind $sto.Kind;
+        Assert-AreEqual $accessTier $sto.AccessTier;
+        Assert-AreEqual $true $sto.Encryption.Services.Blob.Enabled
+        Assert-AreEqual $true $sto.Encryption.Services.File.Enabled
     
         $stotype = 'Standard_RAGRS';
         $accessTier = 'Cool'
-        Set-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Type $stotype -AccessTier $accessTier -StorageEncryption -DisableEncryptionService $encryptionServiceF -EnableEncryptionService $encryptionServiceB -Force
+        Set-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Type $stotype -AccessTier $accessTier -StorageEncryption -Force
         
         $sto = Get-AzureRmStorageAccount -ResourceGroupName $rgname  -Name $stoname;
         $stotype = 'StandardRAGRS';
-        Assert-AreEqual $sto.StorageAccountName $stoname;
-        Assert-AreEqual $sto.Sku.Name $stotype;
-        Assert-AreEqual $sto.Location $loc;
-        Assert-AreEqual $sto.Kind $kind;
-        Assert-AreEqual $sto.AccessTier $accessTier;
-        Assert-AreEqual $sto.Encryption.Services.Blob.Enabled $true
-        Assert-AreEqual $sto.Encryption.Services.File.Enabled $false
+        Assert-AreEqual $stoname $sto.StorageAccountName;
+        Assert-AreEqual $stotype $sto.Sku.Name;
+        Assert-AreEqual $loc $sto.Location;
+        Assert-AreEqual $kind $sto.Kind;
+        Assert-AreEqual $accessTier $sto.AccessTier;
+        Assert-AreEqual $true $sto.Encryption.Services.Blob.Enabled
+        Assert-AreEqual $true $sto.Encryption.Services.File.Enabled
 
         $stotype = 'Standard_GRS';
-        Set-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Type $stotype -StorageEncryption -DisableEncryptionService $encryptionServiceBF
+        Set-AzureRmStorageAccount -ResourceGroupName $rgname -Name $stoname -Type $stotype -StorageEncryption 
         
         $sto = Get-AzureRmStorageAccount -ResourceGroupName $rgname  -Name $stoname;
         $stotype = 'StandardGRS';
-        Assert-AreEqual $sto.StorageAccountName $stoname;
-        Assert-AreEqual $sto.Sku.Name $stotype;
-        Assert-AreEqual $sto.Location $loc;
-        Assert-AreEqual $sto.Kind $kind;
-        Assert-AreEqual $sto.AccessTier $accessTier;
-        Assert-AreEqual $sto.Encryption.Services.Blob.Enabled $false
-        Assert-AreEqual $sto.Encryption.Services.File.Enabled $false
+        Assert-AreEqual $stoname $sto.StorageAccountName;
+        Assert-AreEqual $stotype $sto.Sku.Name;
+        Assert-AreEqual $loc $sto.Location;
+        Assert-AreEqual $kind $sto.Kind;
+        Assert-AreEqual $accessTier $sto.AccessTier;
+        Assert-AreEqual $true $sto.Encryption.Services.Blob.Enabled
```

Comment:
Shouldn't these be taken out now that encryption is permanently enabled? Apply across the board.

Reply:
remove most if this check, only leave 1 test case for the check: Test-SetAzureRmStorageAccountKeySource. Since Keysource is related with Encryption service.